src/lib/react-query/projects.ts

```diff
@@ -442,6 +442,42 @@ export function changeMemberRoleMutationOptions({
 	>
 }
 
+export function removeProjectMemberMutationOptions({
+	projectApi,
+	projectId,
+	queryClient,
+}: {
+	projectApi: MapeoProjectApi
+	projectId: string
+	queryClient: QueryClient
+}) {
+	return {
+		...baseMutationOptions(),
+		mutationFn: async ({ deviceId, reason }) => {
+			// Have to avoid passing `undefined` explicitly
+			// See https://github.com/digidem/rpc-reflector/issues/21
```

Comment:
Would it be better if we also accepted `null` values on the backend and treated them like `undefined`?